src/drivers/distance_sensor/lightware_sf45_serial/lightware_sf45_serial_main.cpp

```diff
@@ -102,7 +102,7 @@ static int usage()
 
 Serial bus driver for the Lightware SF45/b Laser rangefinder.
 
-Setup/usage information: https://docs.px4.io/master/en/sensor/sfxx_lidar.html
+Setup/usage information: https://docs.px4.io/main/en/sensor/rangefinders.html
```

Comment:
I would remove these kind of links from the drivers. Hard to keep them in sync, they cost flash and honestly you should get the idea yourself to type "PX4 SF45" in google when you're blocked.